djangocms_moderation/admin_actions.py

```diff
@@ -148,14 +148,14 @@ def add_items_to_collection(modeladmin, request, queryset):
                 args=(),
             ),
             version_ids=",".join(version_ids),
-            return_to_url=request.META.get("HTTP_REFERER"),
+            return_to_url=request.META.get("HTTP_REFERER", "/"),
```

Comment:
Going by the error I would return an empty string by default: "Did you mean to pass an empty string or omit the value?"

Reply:
Fixed